drop-in/src/main/java/com/adyen/checkout/dropin/ComponentParsingProvider.kt

```diff
@@ -253,16 +261,21 @@ internal fun getComponentFor(
     dropInConfiguration: DropInConfiguration,
     amount: Amount
 ): PaymentComponent<PaymentComponentState<in PaymentMethodDetails>, Configuration> {
-
+    val context = fragment.requireContext()
```

Comment:
Is it possible that we call this before the fragment is actually created or after it's been destroyed?

Reply:
Seeing where we call it from now, it's mostly in fragments inside `onCreate`